content/docs/user-guide/packaging/project-export-cli.md

```diff
@@ -0,0 +1,254 @@
+---
+linkTitle: Project Export CLI
+title: Project Export CLI Tool
+description: Learn how to use the Project Export CLI to automate preparing your project for release.
+toc: true
+weight: 400
+---
+With the new `export-project` CLI command, it is now possible to automate the packaging and release of game projects.
+
+{{< note >}}
+If you wish to learn how to prepare projects manually, please consult the page [Creating a Project Game Release Layout for Windows.](/docs/user-guide/packaging/windows-release-builds)
```

Comment:
This points to a window-specific layout instruction page. That should be part of the note (... for windows)